src/components/note/note.js

```diff
@@ -1,60 +1,98 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import BookImage from '../book-image/book-image';
```

Comment:
What do you think about using the [`book` icon from Assembly](https://labs.mapbox.com/assembly/icons) here instead? I do like the lil book image, but I also like consistency!

Reply:
Added!

![image](https://user-images.githubusercontent.com/2180540/71285017-2556bd80-2332-11ea-8a9f-ad69d990a6b5.png)